source/concepts/environments/abac-resource-restrictions.html.md.erb

```diff
@@ -0,0 +1,104 @@
+---
+owner_slack: "#modernisation-platform"
+title: Resource restrictions based on ABAC
+last_reviewed_on: 2024-07-17
+review_in: 6 months
+---
+
+<!-- Google tag (gtag.js) -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-NXTCMQ7ZX6"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', 'G-NXTCMQ7ZX6');
+</script>
+
+# <%= current_page.data.title %>
+
+## Problem 
+
+The instance-management role is used to provide teams with a limited-access entrypoint into AWS accounts. For example, an application may want to provide support persons with a way to manage instances without having wider permissions. This works well in single-application situations such as our standard offering to customers (one account per application environment).
```

Comment:
"support persons" comes across as odd. I'd just use "support staff" or "the support team"